packages/amplify-cli/src/lib/input-params-manager.js

```diff
@@ -26,10 +26,12 @@ function normalizeKey(key) {
 }
 
 function normalizeValue(key, value) {
-  if (key === 'app') {
```

Comment:
Why are we removing this? This is a part of the latest release for the sample apps I believe. @nikhname could you please review this?

Reply:
Yes, I don't think this should be removed. The workaround below is a general way to do this but it ignores errors in JSON.parse